tasks/prime.py

```diff
@@ -238,7 +238,7 @@ def on_start(self):
         user_id = json_resp["id"]
 
         # Setup customer profile
-        duty_stations = self.client.get(self.customer_path("/internal/duty_stations?search=palms"))
+        duty_stations = self.client.get(self.customer_path("/internal/duty_stations?search=white%20sands"))
```

Comment:
I tested this out and it works like you said.  I have a possibly hacky idea, which you should feel free to shoot down.

I see KKFA has 2 duty stations "Fort McCoy" that provides services counseling and "McConnel AFB" that would skip it and go to the TOO.  If we randomly picked one of those 2 and they are the first 2 results when searching for "McCo" then we would get seed data in the Services Counselor and TOO office queues 🤔.

Unfortunately we don't have a Services Counselor task that actually submits the move to the TOO, which is quite simple if the move is in the complete state.  We could also wait for that to be written.

Reply:
Nevermind I've already forgotten how the new TOO queue works, you'd have to override the postal code of the shipment as well to match.

Reply:
This endpoint has changed too. It's now `/duty_locations` instead of `/duty_stations`

Reply:
Just updated! Depending if there are additional field name changes we may have to make some additional changes.